shared/actions/profile/proofs.js

```diff
@@ -72,6 +72,9 @@ function _addProof(action: ProfileGen.AddProofPayload) {
       actions.push(Saga.put(_registerBTC()))
       break
     // flow needs this for some reason
+    case 'web':
+      actions.push(Saga.call(_addServiceProof, 'web'))
```

Comment:
don't think you need to special-case this, you could just `case 'web':` below `case 'https':`